build.sh

```diff
@@ -0,0 +1,114 @@
+#!/bin/bash
+cd `dirname ${0}`
+
+function printHelpAndExit () {
+    echo ""
+    echo "    ./build.sh [options] <version>"
+    echo "       Options are:"
+    echo "         -h Print help and exit"
+    echo "         -m Provide kio description from command line instead of running editor"
+    echo ""
+    exit -1
+}
+
+KIO_MESSAGE=""
+while [[ "x$1" != "x" ]] && [[ ${1:0:1} == "-" ]]; do
+    case $1 in
+        -h|--help)
+            printHelpAndExit
+        ;;
+        -m|--message)
+            shift
+            KIO_MESSAGE="$1"
+        ;;
+        *)
+            echo "Unsupported option ${1}" &>2
+            printHelpAndExit
+        ;;
+    esac
+    shift
+done
+set -u -f
+
+if [[ ! $# -eq 1 ]] ; then
+    echo "Version is not specified or too much data" &>2
+    printHelpAndExit
+fi
+
+pierone login
+if [ $? -ne 0 ]; then
+    echo "Failed to login to pierone" &>2
+    exit 2
+fi
+
+APPLICATION_ID="nakadi"
+DOCKER_IMAGE_TAG="pierone.stups.zalan.do/aruha/${APPLICATION_ID}"
```

Comment:
Yes, forgot to move it to configuration.
The problem is that we are trying to create this script in public repo instead of moving senza.yaml to private repo